shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/network/MealApi.kt

```diff
@@ -42,9 +42,9 @@ class MealApi {
 
     companion object {
         // iOS can use this endpoint
-//        private const val MEALS_ENDPOINT = "http://localhost:3000/Meals"
+        private const val MEALS_ENDPOINT = "http://localhost:3000/Meals"
 
         // Android needs to switch to this endpoint
-        private const val MEALS_ENDPOINT = "http://10.0.2.2:3000/meals"
+//        private const val MEALS_ENDPOINT = "http://10.0.2.2:3000/meals"
```

Comment:
This pickled my brain - thought I had broke something when nothing was showing 😬 
I'm gonna fix this after the demo to be a switch on platform type

Reply:
Nice idea